src/Posts.jsx

```diff
@@ -45,68 +45,50 @@ const selectTab = (selectedTab) => {
   loadMorePosts();
 };
 
-let gatewayModeratedUsersRaw = Social.get(
-  `${moderatorAccount}/moderate/users`,
-  "optimistic",
-  {
-    subscribe: true,
-  }
-);
-
+// get the full list of posts that the current user has flagged so
+// they can be hidden
 const selfFlaggedPosts = context.accountId
   ? Social.index("flag", "main", {
       accountId: context.accountId,
     })
   : [];
 
-const selfModeration = context.accountId
-  ? Social.index("moderate", "main", {
-      accountId: context.accountId,
-    })
-  : [];
-
-if (gatewayModeratedUsersRaw === null) {
```

Comment:
Global moderation is now handled through QueryApi. See PR https://github.com/near/near-discovery-components/tree/375-filter-activityfeed-based-on-new-moderation-format